task/activity.go

```diff
@@ -15,6 +15,7 @@ type callActivityOption func(*callActivityOptions) error
 type callActivityOptions struct {
 	rawInput    *wrapperspb.StringValue
 	retryPolicy *RetryPolicy
+	AppID       string
```

Comment:
This must be optional.

```suggestion
	AppID       *string
```